packages/acceptance-tests/pkg-tests-core/sources/utils/makeTemporaryEnv.ts

```diff
@@ -31,16 +31,16 @@ global.makeTemporaryEnv = generatePkgDriver({
       ? [projectFolder]
       : [];
 
-    const res = await execFile(process.execPath, [`${__dirname}/../../scripts/run-yarn.js`, ...cwdArgs, command, ...args], {
+    const res = await execFile(process.execPath, [`${__dirname}/../../../../../scripts/run-yarn.js`, ...cwdArgs, command, ...args], {
       cwd: cwd || path,
       env: {
         [`HOME`]: tempHomeFolder,
         [`USERPROFILE`]: tempHomeFolder,
         [`PATH`]: `${nativePath}/bin${delimiter}${process.env.PATH}`,
         [`TEST_ENV`]: `true`,
         [`YARN_GLOBAL_FOLDER`]: `${nativePath}/.yarn/global`,
-        [`YARN_NPM_REGISTRY_SERVER`]: registryUrl,
-        [`YARN_UNSAFE_HTTP_WHITELIST`]: new URL(registryUrl).hostname,
+        [`YARN_NPM_REGISTRY_SERVER`]: registryUrl!,
```

Comment:
There should always be a `registryUrl`, is the `!` needed?

Reply:
Fixed